charts/k8s-monitoring/templates/destinations/_destination_otlp.tpl

```diff
@@ -274,10 +274,10 @@ otelcol.processor.filter {{ include "helper.alloy_name" .name | quote }} {
 {{- end }}
   }
 {{- end }}
-{{- if and .logs.enabled .processors.filters.logs.log_record }}
+{{- if and .logs.enabled .processors.filters.logs.logRecord }}
   logs {
-    log_record = [
-{{- range $filter := .processors.filters.logs.log_record }}
+    logRecord = [
```

Comment:
```suggestion
    log_record = [
```
This one has to stay, since it corresponds to this Alloy config:
https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.filter/#logs-block